components/bsx/Create/CreateToken.vue

```diff
@@ -261,7 +261,7 @@ export default class CreateToken extends mixins(
     return !this.listed || (price > 0 && price <= this.maxPrice)
   }
 
-  protected async submit(): Promise<void> {
+  protected async submit(retryCount = 0): Promise<void> {
```

Comment:
why do we actually need this parameter here?

Reply:
Because it needs an initial value to limit the retry time. Or it will retry forever

Reply:
I limited it to three retries

Reply:
Ahh whoops, didn't notice that the function is calling itself there.